src/aics-image-viewer/components/CellViewerCanvasWrapper/index.tsx

```diff
@@ -111,6 +111,7 @@ const STYLES: Styles = {
   view3d: {
     width: "100%",
     display: "flex",
+    overflow: "hidden",
```

Comment:
fixes an unrelated issue I've run into on Firefox with scroll bars appearing in the viewport